acceptance/lib/helper.rb

```diff
@@ -30,7 +30,8 @@ def self.initialize_config(options)
     puppet_build_version = get_option_value(options[:puppet_build_version],
                          nil, "Puppet Agent Development Build Version",
                          "PUPPET_BUILD_VERSION",
-                         "ba39f2f2fd1b4def9a7f366d469acb1c5f6db824", :string)
+                         "1a2a63d86e8f7ee6cc354b988f49b36d007620f4",
```

Comment:
Here, we might want 5cb5a63982c01f3ed0182c25b61213d208573d8d.  See `SUITE_COMMIT` for https://jenkins.puppetlabs.com/view/puppet-agent/view/master/view/puppet-agent/job/platform_puppet-agent_intn-van-sys_suite-daily-puppet-master/178/.